examples/game/breakout.rs

```diff
@@ -76,24 +78,84 @@ struct Collider;
 #[derive(Component)]
 struct Brick;
 
+// This bundle is a collection of the components that define a "wall" in our game
+#[derive(Bundle)]
+struct WallBundle {
+    collider: Collider,
+    // You can nest bundles inside of other bundles like this
+    // Allowing you to compose their functionality
+    #[bundle]
+    sprite_bundle: SpriteBundle,
+}
+
+/// Which side of the arena is this wall located on?
+enum WallLocation {
+    Left,
+    Right,
+    Bottom,
+    Top,
+}
+
+impl WallBundle {
+    // This "builder method" allows us to reuse logic across our wall entities,
+    // making our code easier to read and less prone to bugs when we change the logic
+    fn new(orientation: WallLocation) -> WallBundle {
+        // This allows us to just use `Left`, rather than `WallOrientation::Left` everywhere
+        use WallLocation::*;
+
+        let translation = match orientation {
+            Left => Vec2::new(-PLAY_AREA_BOUNDS.x / 2.0, 0.0),
+            Right => Vec2::new(PLAY_AREA_BOUNDS.x / 2.0, 0.0),
+            Bottom => Vec2::new(0.0, -PLAY_AREA_BOUNDS.y / 2.0),
+            Top => Vec2::new(0.0, PLAY_AREA_BOUNDS.y / 2.0),
+        }
+        // We need to convert our Vec2 into a Vec3, by giving it a z-coordinate
+        .extend(0.0);
+
+        let scale = match orientation {
+            Left => Vec2::new(WALL_THICKNESS, PLAY_AREA_BOUNDS.y + WALL_THICKNESS),
+            Right => Vec2::new(WALL_THICKNESS, PLAY_AREA_BOUNDS.y + WALL_THICKNESS),
+            Bottom => Vec2::new(PLAY_AREA_BOUNDS.x + WALL_THICKNESS, WALL_THICKNESS),
+            Top => Vec2::new(PLAY_AREA_BOUNDS.x + WALL_THICKNESS, WALL_THICKNESS),
+        }
+        .extend(1.0);
```

Comment:
A comment here like the one above would be great. New users often trip over z scale in 2d.

(Or maybe this example should use `custom_size` instead of scale... hm.)